README.md

```diff
@@ -113,6 +113,8 @@ The API reference can be found in `target/doc/amethyst/index.html`.
 
 ## Questions / Help
 
+We do not support anything other than the most recent Rust stable release.  Use nightly and beta channels with this project at your own risk.
```

Comment:
Maybe we should say "We do not explicitly support"? Also, there are two spaces between the two sentences.